src/module.ts

```diff
@@ -124,32 +118,39 @@ class Ctrl extends MetricsPanelCtrl {
       return;
     }
     try {
-      var items = this.mapper.mapMetricData(this._seriesList);
+      // TODO: set this.items also
+      this.items = this.mapper.mapMetricData(this._seriesList);
     } catch(e) {
       this._panelAlert.active = true;
       this._panelAlert.message = ERROR_MAPPING;
       return;
     }
     if(this._panelConfig.getValue('sortingOrder') === 'increasing') {
-      items = _.sortBy(items, i => i.aggregatedProgress);
+      this.items = _.sortBy(this.items, i => i.aggregatedProgress);
     }
     if(this._panelConfig.getValue('sortingOrder') === 'decreasing') {
-      items = _.sortBy(items, i => -i.aggregatedProgress);
+      this.items = _.sortBy(this.items, i => -i.aggregatedProgress);
     }
-    this.$scope.items = items;
+    this.$scope.items = this.items;
 
     if(this._tooltip !== undefined) {
       this._tooltip.destroy();
     }
-    this._tooltip = new GraphTooltip(
-      () => this._seriesList, items, this.panel.tooltipMode
-    );
+    this._tooltip = new GraphTooltip(this.panel.tooltipMode);
     this._panelAlert.active = false;
 
   }
 
-  onHover(index: number, event: any, title?: any, value?: any) {
-    this._tooltip.show(event.originalEvent, index, title, value);
+  onHover(event: { index: number, event: any }, title?: any, value?: any) {
```

Comment:
```suggestion
  onHover(event: { index: number, event: any }) {
```
looks like we don't use them anymore